src/StudyUtils.in.jsm

```diff
@@ -289,6 +587,21 @@ class StudyUtils {
       this._telemetry({study_state: "installed"}, "shield-study");
     }
   }
+
+  /**
+  * @async
+  * Ends the study:
+  *  - Removes the study from the active list of telemetry experiments
+  *  - Opens a new tab at a specified URL, if present (e.g. for a survey)
+  *  - Sends a telemetry ping about the nature of the ending
+  *    (positive, neutral, negative)
+  *  - Sends an exit telemetry ping
+  * @param {string} reason - The reason the study is ending, see
+  * schema.studySetup.json
+  * @param {string} fullname -  optional, the full name of the study state,
+  * see schema.studySetup.json
```

Comment:
I'm not sure these two `@param` entries are correct. It looks like `endStudy()` takes a single Object parameter w/ two keys, not 2 distinct, named parameters.

Reply:
You're right; I've updated this for the destructured object argument.